include/OpenSpaceToolkit/Astrodynamics/Trajectory/State.hpp

```diff
@@ -78,11 +81,31 @@ class State
     /// @brief                  Utility constructor for Position/Velocity only.
     ///
     /// @param                  [in] anInstant An instant
-    /// @param                  [in] aPosition The cartesian position at the instant in International System of Units
-    /// @param                  [in] aVelocity The cartesian velocity at the instant in International System of Units
+    /// @param                  [in] aPosition The Cartesian position at the instant
+    /// @param                  [in] aVelocity The Cartesian velocity at the instant
 
     State(const Instant& anInstant, const Position& aPosition, const Velocity& aVelocity);
 
+    /// @brief                  Utility constructor for Position/Velocity/Attitude only.
+    ///
+    /// @param                  [in] anInstant An instant
+    /// @param                  [in] aPosition The Cartesian position at the instant
+    /// @param                  [in] aVelocity The Cartesian velocity at the instant
+    /// @param                  [in] anAttitude The attitude at the instant in International System of Units
```

Comment:
So ref your question, I think it's common to use the body-fixed convention, i.e. the quaternion used here in the State should represent the rotation required to go from `FrameOfReference` (ECEF, ECI) to Satellite `BodyFrame`, lmk if you agree (in the future we can add different convention types if needed but definitely not needed now)

Reply:
Resolving